util/Nginx/proxy.conf

```diff
@@ -4,12 +4,12 @@ proxy_set_header        X-Real-IP         $remote_addr;
 proxy_set_header        X-Forwarded-For   $proxy_add_x_forwarded_for;
 proxy_set_header        X-Url-Scheme      $scheme;
 proxy_set_header        X-Forwarded-Proto $scheme;
-client_max_body_size    105m;
+client_max_body_size    505m;
```

Comment:
Is this Nginx config the same between cloud and self-hosted?

I think this might be the first instance where we might want to delineate between the two. There isn't anything in cloud that accepts/expects anything near 500MB requests.

Reply:
I checked with @joseph-flinn and he said that we don't use nginx in the cloud. We might start using it down the line if/when we migrate to K8, but for now it's just self-hosted, so this should be fine.